package.json

```diff
@@ -25,7 +25,7 @@
     "eslint-config-airbnb": "^15.0.1",
     "eslint-loader": "^1.6.1",
     "eslint-plugin-import": "^2.2.0",
-    "eslint-plugin-jsx-a11y": "^6.0.2",
+    "eslint-plugin-jsx-a11y": "^5.1.1",
```

Comment:
@jeffrey-xiao you can remove this portion from the PR, @sjhworkman separated this into another PR that is now merged